examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx

```diff
@@ -17,7 +17,7 @@ export default function Home() {
     currentNum: null as null | Field,
     publicKey: null as null | PublicKey,
     zkappPublicKey: null as null | PublicKey,
-    creatingTransaction: false
+    creatingTransaction: false,
```

Comment:
Could you clarify whether these formatting changes cover every tutorial or are just specific to these files changed in the PR? If it's only these files, could you apply the same formatting to all remaining files? Thanks!

Reply:
Agreed. Are theses just prettier formatting updates?

Reply:
Reverted.